MNE/disp3D/engine/model/items/network/networktreeitem.cpp

```diff
@@ -238,7 +238,7 @@ void NetworkTreeItem::plotNetwork(const Network& tNetworkData, const QVector3D&
 
         //Plot out edges
         for(int j = 0; j < lNetworkNodes.at(i)->getEdgesOut().size(); ++j) {
-            if(abs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
+            if(fabs(lNetworkNodes.at(i)->getEdgesOut().at(j)->getWeight()) >= vecThreshold.x()) {
```

Comment:
Use std::fabs or std::abs instead, because using abs() or fabs() calls the old C functions.  abs() is not overloaded for float types, std::abs() is. So everytime abs(3.0) is called its gets converted to abs(3). This error is multiple times in this file and probably in others aswell.